guides/source/active_record_encryption.md

```diff
@@ -141,199 +252,233 @@ When enabled, all the encryptable attributes will be encrypted according to the
 
 #### Action Text Fixtures
 
-To encrypt Action Text fixtures, you should place them in `fixtures/action_text/encrypted_rich_texts.yml`.
+To encrypt Action Text fixtures, you can place them in `fixtures/action_text/encrypted_rich_texts.yml`.
 
-### Supported Types
+### Encoding
 
-`active_record.encryption` will serialize values using the underlying type before encrypting them, but, unless using a custom `message_serializer`, *they must be serializable as strings*. Structured types like `serialized` are supported out of the box.
+When encrypting strings non-deterministically, their original encoding is preserved automatically.
 
-If you need to support a custom type, the recommended way is to use a [serialized attribute](https://api.rubyonrails.org/classes/ActiveRecord/AttributeMethods/Serialization/ClassMethods.html). The declaration of the serialized attribute should go **before** the encryption declaration:
+For deterministic encryption, Rails stores the string encoding alongside the ciphertext. However, to ensure consistent encryption output—especially for querying or enforcing uniqueness—the library forces UTF-8 encoding by default. This avoids producing different ciphertexts for identical strings with different encodings.
```

Comment:
Nitpick, I think commas help with readability, but feel free to ignore if you disagree.

```suggestion
For deterministic encryption, Rails stores the string encoding alongside the ciphertext. However, to ensure consistent encryption output, especially for querying or enforcing uniqueness, the library forces UTF-8 encoding by default. This avoids producing different ciphertexts for identical strings with different encodings.
```